setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
The 0.8 version of pyBarcode introduced code128 support and I really want this support into our implementation. [Latest PyPI version is 0.8b1](https://pypi.python.org/pypi?%3Aaction=search&term=pyBarcode&submit=search) but if I put the `>=0.8` requirement the beta version is not retrieved:

```
pip install 'pyBarcode>=0.8'
Collecting pyBarcode>=0.8
  Could not find a version that satisfies the requirement pyBarcode>=0.8 (from versions: 0.5.1, 0.6, 0.7, 0.8b1)
No matching distribution found for pyBarcode>=0.8
```

The real solution is to make pyBarcode release a v0.8 tag and push it on PyPI but the [pyBarcode repository seems quite inactive right now](https://bitbucket.org/whitie/python-barcode). I emailed the maintainer but had no response so far ; maybe we should fork it and rename it?